modules/launcher/src/main/resources/broker.yaml

```diff
@@ -60,6 +60,19 @@ ballerina.broker.auth:
     # Optional properties
     properties:
      loginModule: io.ballerina.messaging.broker.auth.authentication.jaas.UserStoreLoginModule
+ # Broker authorization related configurations.
+ authorization:
+  # Enable the authorization
+  enabled: true
+  authProvider:
+   # AuthProvider implemetation
+   className: io.ballerina.messaging.broker.auth.authorization.provider.UserStoreAuthProvider
+   properties:
+  cache:
+   # Cache timeout in Minutes
+   timeout: 15
+   # Maximum Cache size
+   size: 5000
```

Comment:
1. Shall we specify the unit?
2. We also need to document these values in config document.

Reply:
Fixed in d43ea86